package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "mocha tests/lint-test",
-    "test": "mocha tests --recursive"
+    "test": "mocha tests node-tests --recursive"
```

Comment:
We generate into the node-tests folder, where the blueprint-tests are. It won't pick them up otherwise.